### DISCUSSION TITLE
Best Practice to warm up an ip address - 50.000 Mails/day

### DISCUSSION THREAD

**Discussion by Pacerino** [postalserver/postal#2680] [Help with using Postal] [answered]
I work for a company that sends about 1 million emails per month. In the past, there was no employee to set up the e-mail traffic correctly, so we simply used sendmail to send e-mails. It is a miracle that our mails/IP have not yet ended up on a blacklist.

In the meantime, I have made sure that we have our own dedicated server with a dedicated IP address. I have successfully set up Postal there in the past few days and made all the configurations such as SPF, DKIM, DMARC etc. A small number of "test mails" no longer end up in the spam folders of recipients (tested with Google, Outlook and Yahoo).

My question now is, is there a good solution to automatically warm up the IP? I have seen that Postal offers this option using IP pools, but we currently only have one IP address and only intend to send via one. In addition, we send important e-mails that have to reach the recipient on a daily basis. Unfortunately, simply delaying emails is not possible, as our emails contain important offers for short-term temporary work for our customers. This means that all recipients must necessarily receive the mail. 

Switching to a SaaS provider would be possible, but would go beyond the price range available to me.

Would multiple IP addresses be a solution? There are almost 50,000 mails a day, if you distribute them to 4 different IP addresses and thus send 12,500 mails per IP address, would that be a good solution?

Many thanks for your help!

**Comment by willpower232** [comment, marked as answer]
Well if you haven't changed IPs since the sendmail time then it probably doesn't need warming up any more :sweat_smile: 

Warming up IP addresses would always involve some amount of slowing down the emails going through the IP address so if you're able to space out the less time-sensitive ones and monitor how well they're sending then it should be fine.

If you send a test message to https://www.mail-tester.com/ and there are no causes for concern, I would imagine you are fine.

**Reply by Pacerino** [reply]
Thank you for the answer! Actually the IP address has changed. I've ran the test suggested by you and we're getting 10/10 points for the domain running on Postal. I'll take a look into the warm up practice, besides that, does Postal offer the option to release mails manually?

I appreciate your help!

**Reply by willpower232** [reply]
Unfortunately there isn't a bulk button to release the emails, someone mentioned somewhere else about running an SQL to queue up all the held messages. I usually just open all the tabs and click the release button :sweat_smile:
